### ISSUE TITLE
Execution failed for task ':launcher:checkReleaseDuplicateClasse' 

### ISSUE THREAD

**Issue by nintendaii** [qonversion/unity-sdk#210]
Hey. I am new to Quonversion. I have tried to import the SDK following [this](https://documentation.qonversion.io/docs/unity-sdk) guide (except [multidex](https://developer.android.com/studio/build/multidex#mdex-gradle) step) and then tried to build the project but I have got this error. I already had a Unity IAP implemented so I also excluded the `billingclient` dependency.
**Unity: 2021.2.10f1
Qonversion: 7.0.0**
Here are some logs:
`
Execution failed for task ':launcher:checkReleaseDuplicateClasses'.
Could not resolve all files for configuration ':launcher:releaseRuntimeClasspath'.
Failed to transform moshi-1.14.0.jar (com.squareup.moshi:moshi:1.14.0) to match attributes {artifactType=android-classes-jar, org.gradle.category=library, org.gradle.dependency.bundling=external, org.gradle.jvm.environment=standard-jvm, org.gradle.jvm.version=8, org.gradle.libraryelements=jar, org.gradle.status=release, org.gradle.usage=java-runtime, org.jetbrains.kotlin.platform.type=jvm}.
       Execution failed for JetifyTransform: /Users/daniel/.gradle/caches/modules-2/files-2.1/com.squareup.moshi/moshi/1.14.0/df173a3bfa0bd9e0cd2a8fc14b24fc336dca0de0/moshi-1.14.0.jar.
         > Failed to transform '/Users/daniel/.gradle/caches/modules-2/files-2.1/com.squareup.moshi/moshi/1.14.0/df173a3bfa0bd9e0cd2a8fc14b24fc336dca0de0/moshi-1.14.0.jar' using Jetifier. Reason: IllegalArgumentException, message: Unsupported class file major version 60. (Run with --stacktrace for more details.)`

<img width="254" alt="Screenshot 2024-02-27 at 16 08 45" src="https://github.com/qonversion/unity-sdk/assets/44233090/056c5a5a-bc81-4752-99c9-4c4ce04c8ce8">

**Comment by SpertsyanKM** [comment]
Hello! 

The `Unsupported class file major version 60.` error usually means inconsistency with the JDK version. Which version do you use?

**Comment by nintendaii** [comment]
Hey, thanks for the reply. I am using JDK version that comes with Unity by default
<img width="474" alt="Screenshot 2024-02-27 at 17 30 48" src="https://github.com/qonversion/unity-sdk/assets/44233090/76ee5133-d216-477a-aa44-e3f4ac93a407">

**Comment by nintendaii** [comment]
I assume this is version 8 since I am using Unity 2021.2.10
<img width="781" alt="Screenshot 2024-02-27 at 17 32 36" src="https://github.com/qonversion/unity-sdk/assets/44233090/d8b73984-370a-4b22-b6ba-7d52a39a6849">

**Comment by SpertsyanKM** [comment]
We will need some time to dig deeper into this issue. I'll come back as soon as I have any news.

**Comment by nintendaii** [comment]
Sure, thanks!

**Comment by nintendaii** [comment]
Hey @SpertsyanKM. Any update on this issue?

**Comment by SpertsyanKM** [comment]
Hello, @nintendaii. We understand the problem criticality for you, but we will need more time for investigation due to high level of support requests in a while. Sorry for that. We will try our best to return as soon as possible.

**Comment by SpertsyanKM** [comment]
Hello, @nintendaii.

Thank you for your patience and for bringing this issue to our attention. I sincerely apologize for the delay in getting back to you.

I've reviewed the issue you reported, but unfortunately, I was unable to reproduce it on my end. I've created a bare project on the minimally supported Unity version - 2021.3, followed the documentation steps to import our SDK and it worked as expected. The difference is that my Unity installation uses JDK 11, the [supported version](https://docs.unity3d.com/Manual/android-sdksetup.html) for Unity 2021.3. 

If your issue is still relevant, have you tried to upgrade your Unity version to 2021 or higher?
